supporting_scripts/analysis-of-endpoint-connections/analysisOfEndpointConnections.config.yml

```diff
@@ -0,0 +1,279 @@
+  - 'src/main/webapp/app/exercises/programming/manage/services/build-plan.service.ts'
+  - 'src/main/webapp/app/exam/manage/exams/exam-import/exam-import-paging.service.ts'
+
+excludedRestCalls:
+  # The Endpoints matching the following REST calls are provided by SpringBootActuator and spring,
+  # see src/main/resources/config/application.yml management.endpoints.web.exposure.include
+  - 'management/configprops'
+  - 'management/env'
+  - '/management/inf'
+  - 'management/info'
+  - 'management/health'
+  - 'management/loggers'
+  - 'management/loggers/${name}'
+  - 'management/jhimetrics'
+  - 'management/threaddump'
+
+  # The following Endpoints are all from this file: course-management.service.ts
+  # The Endpoint on the server looks like this: 'api/courses/{courseId}/instructors'
+  # or this: api/courses/{courseId}/tutors etc.
+  # Issue: #9372 and #9371
+  - 'api/courses/${courseId}/${courseGroup}'
+  - 'api/courses/${courseId}/${courseGroup}/${login}'
+
+  # The following Endpoints are all from this file: quiz-exercise.service.ts
+  # The Endpoint on the server looks like this: 'api/quiz-exercises/{quizExerciseId}/{action}'
+  - 'api/quiz-exercises/${quizExerciseId}/open-for-practice'
+  - 'api/quiz-exercises/${quizExerciseId}/start-now'
+  - 'api/quiz-exercises/${quizExerciseId}/end-now'
+  - 'api/quiz-exercises/${quizExerciseId}/set-visible'
+
+  # These 3 files are part of the file: iris-chat-http.service.ts
+  - 'api/iris/sessions/${sessionId}/messages/${messageId}/helpful/${helpful}'
+  - 'api/iris/${identifier}/sessions/current'
+  - 'api/iris/${identifier}/sessions'
+
+  - 'api/public/lti13/auth-login'
+  - '${this.resourceUrl}'
+  # The following Endpoints are all from this file: course-exercise.service.ts
+  - '${this.resourceUrl}/${courseId}/file-upload-exercises'
+  - '${this.resourceUrl}/${courseId}/text-exercises'
+  - '${this.resourceUrl}/${courseId}/modeling-exercises'
+  - '${this.resourceUrl}/${courseId}/programming-exercises'
+
+  # Issue: #9372
+  - 'api/programming-exercises/${targetExerciseId}/static-code-analysis-categories/import'
+
+  # Edge case in exam-management.service.ts
+  - '${url}'
+
+  # Edge cases with multiple possible values in athena.service.ts
+  # Issue: #9371
+  - 'api/athena/${exercise.type}-exercises/${exercise.id}/submissions/${submissionId}/feedback-suggestions'
+  - 'api/athena/courses/${courseId}/${exercise.type}-exercises/available-modules'
+
+  # POSSIBLY UNUSED REST CALLS
+  - 'api/courses/${courseId}/posts/similarity-check'
+  - 'public/content/${filename}'
+  - 'api/courses/${courseId}/results'
+  - 'api/exercises'
+  - 'api/results/${resultId}'
+  - 'api/attachments'
+  - 'api/repository/${participationId}/file'
+
+endpointParsingResultPath: 'endpoints.json'
+restCallParsingResultPath: 'restCalls.json'
+endpointAnalysisResultPath: 'endpointAnalysisResult.json'
+restCallAnalysisResultPath: 'restCallAnalysisResult.json'
+clientDirPath: 'src/main/webapp/app'
```

Comment:
Since we have some (potentially long) lists above these simple config values, I'd suggest moving them to the top of the file